include/coruja/container/view/transform.hpp

```diff
@@ -120,6 +122,7 @@ class coruja_transform_view
     { return {_transform, std::forward<F>(f)}; }
     
 public:
+    using value_type = typename std::decay<coruja::result_of_t<Transform(ranges::range_reference_t<Rng>)>>::type;
```

Comment:
We can't use a `std::decay` here because the transform can return a reference. In this case `std::is_reference<Rng::value_type>::value` should be equal to `true`.